src/mcp_server_sql_analyzer/server.py

```diff
@@ -0,0 +1,247 @@
+import sqlglot
+from pydantic import BaseModel
+from typing import Literal
+from sqlglot import parse_one, exp
+from sqlglot.expressions import Expression
+from sqlglot.optimizer.scope import build_scope
+from mcp.server.fastmcp import FastMCP
+
+mcp = FastMCP("SQL Analyzer")
+
+
+class ParseResult(BaseModel):
+    isValid: bool
```

Comment:
for consistency with the snake-case variables (`read_dialect`, `write_dialect`), what do you think about calling this `is_valid`?

Reply:
Great catch!